### DISCUSSION TITLE
Add a private project to a private repository on a free tier organization.

### DISCUSSION THREAD

**Discussion by CodingAleCR** [community/community#127294] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I've been trying to convert items into issues inside a private project that belongs to an organization with the free tier (I'm the owner). However, it doesn't seem to recognize the projects of the organization. The project that I want to make issues for is a private project (not sure if that helps). Is this something that's even possible?

**Comment by ebndev** [comment]
Hi @CodingAleCR,

Have you invited yourself to the organization and as a collaborator on those repos?

**Comment by bennefeldtim** [comment]
Possible Issues and Fixes

Organization Restrictions:
If your organization has restrictions on project interactions, you might need to adjust these settings. Go to your organization's settings, and under "Projects", ensure the relevant permissions are enabled.

Repository Linkage:
Ensure that the private project is correctly linked to the repository. If the project is not correctly associated with a repository, GitHub won't be able to convert items into issues.

Permission Levels:
Verify that your user role within the organization has sufficient permissions to create issues. Even as an owner, you might need to adjust specific repository or project settings.

**Comment by hoanganh6789** [comment]
Hi

**Comment by CodingAleCR** [comment, marked as answer]
Oh I forgot about this. I found what the issue was. 

So my GitHub code project had the "Issues" tab disabled. That's why I wasn't able to convert the items into issues in my project.

**Reply by ebndev** [reply]
Thanks for coming back to provide the answer!
